pkg/tsdb/graphite/graphite.go

```diff
@@ -248,15 +249,25 @@ func (s *Service) toDataFrames(response *http.Response) (frames data.Frames, err
 			values = append(values, value)
 		}
 
+		tags := make(map[string]string)
+		for name, value := range series.Tags {
+			switch value := value.(type) {
+			case string:
+				tags[name] = value
+			case float64:
```

Comment:
Did a small test here: https://play.golang.org/p/Xfx2cQAVMNF, with an interface and 2, seems the value type would be int. I might missed something.

Reply:
@ying-jeanne in practice we should never get an `int` here and always a `float64` or a `string`. These values are coming from unmarshalled JSON which according to this [spec](https://pkg.go.dev/encoding/json#Unmarshal) always uses `float64` for numbers. I've added a unit test to cover 3 cases (string, integer and float in JSON).

Reply:
fair enough :) approved.